scripts/format_new.py

```
@@ -0,0 +1,170 @@
+    from constants import CONTACT_STORE_FILE, OUTPUT_DIR, TMP_PATH
+    from utils import format_date_to_iso, is_date_in_future
+
+# Constants
+EXPORT_PATH = Path("/Users/jessicajohnson/imessage-export")
+
+
+@dataclass
+class Contact:
+    """Contact information."""
+    name: str
+    phone: Optional[str] = None
```

Ruff (0.8.2)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)